### PR TITLE
AtomEmbedMarkup and AtomEmbedUrl migration 3/3

### PR DESCRIPTION
**Pull request by shtukas**
## What does this change?

Third part of the renaming of 

```
InteractiveMarkupBlockElement
InteractiveUrlBlockElement
```

Into

``` 
AtomEmbedMarkupBlockElement
AtomEmbedUrlBlockElement
```

Follow up of: 
1. https://github.com/guardian/dotcom-rendering/pull/1398, and 
2. https://github.com/guardian/frontend/pull/22526

### CODE REVIEW

**Comment by shtukas** [comment]
@oliverlloyd that would be great !

**Comment by shtukas** [comment]
Tested on CODE:

Screenshot 2020-04-24 at 11 32 30

**Comment by oliverlloyd** [comment]
> Should be fairly simple to do this

Why do I make statements like this?

Anyway, so adding tests for elements rendered inside iframes was not easy but is now working and is part of this PR. I created a new 'elements' spec file so we have a place to put all in body element focussed tests so we can now extend the same test method for web iframes once we have support there.

**Comment by gtrufitt** [comment]
> Anyway, so adding tests for elements rendered inside iframes was not easy but is now working and is part of this PR. I created a new 'elements' spec file so we have a place to put all in body element focussed tests so we can now extend the same test method for web iframes once we have support there.

+1 This is great to have!

File: cypress/integration/e2e/article.elements.spec.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                    // chaining more Cypress commands, like ".find(...)"
+                    // https://on.cypress.io/wrap
+                    .then(cy.wrap)
+            );
+        };
+
+        it('should render the corona embed', function() {
+            // https://api.nextgen.guardianapps.co.uk/embed/atom/interactive/interactives%2F2020%2F03%2Fcovid-19-uk%2Fdefault#amp=1
+            cy.server();
+            // Listen for share count call
```
Comment (gtrufitt) [inline cypress/integration/e2e/article.elements.spec.js:27]: Is this the wrong comment?

File: cypress/integration/e2e/article.elements.spec.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            cy.route({
+                method: 'GET',
+                url: '**/interactives/**/covid-19-uk/**',
+            }).as('getCoronaEmbed');
```
Comment (gtrufitt) [inline cypress/integration/e2e/article.elements.spec.js:31]: Don't think you need the alias?

Comment (gtrufitt) [reply cypress/integration/e2e/article.elements.spec.js:31]: What's this route listener actually for?

Comment (oliverlloyd) [reply cypress/integration/e2e/article.elements.spec.js]: > What's this route listener actually for?

Thank you. I completely missed that I wasn't using this anymore!

**Pull request merged**